docs/appendices/release-notes/unreleased.rst

```diff
@@ -93,6 +93,9 @@ Changes
 
 - Added support of ``GROUP BY`` on ``ARRAY`` typed columns.
 
+- Updated the Admin UI to version 1.22.2. It includes a fix for a rendering
```

Comment:
should this not be in fixes 😬 ?

Reply:
Right, we want to include this at the next hotfix releases. Only entries at the fixes section should be backported.

Reply:
Thank you. I've just updated the patch.